### ISSUE TITLE
Text formatting to increase readability

### ISSUE THREAD

**Issue by varshika03** [gautamkrishnar/socli#119]
It would be great if the displayed result on the terminal looks very similar to the original one on the web page. Rather than printing all the text inside the 'post-text' class, there can be distinctions for different html tags like highlighting for <code> tag or blue and underlined for hyperlinks. 
Formatting for other HTML tags can also be thought of so as to make the output more appealing.

**Comment by gautamkrishnar** [comment]
:grinning: thanks a lot @varshika03 looks like a good feature .

**Comment by Rohithgilla12** [comment]
Hey, is this still open i a ready to take it up :+1:

**Comment by gautamkrishnar** [comment]
@Rohithgilla12 thanks for your interest. This one is already assigned to @varshika03.

**Comment by Rohithgilla12** [comment]
Hey itsk dude, I will take up another issue in the future :)

**Comment by virajvchavan** [comment]
For better readability, any code should be highlighted. Better yet, syntax highlighting for the code will be really useful.

**Comment by hedythedev** [comment]
We could use [pygments](https://github.com/pygments/pygments) 👍🏻

**Comment by LeoHop** [comment]
Hi @gautamkrishnar! If this issue is still open I would love to work on it if possible!

**Comment by gautamkrishnar** [comment]
@LeoHop yes you can work on this. Please let us know how you're planning to implement it.

**Comment by calkerns** [comment]
@gautamkrishnar Do you know if LeoHop ever made progress on this? If not, I can try to take a stab at it within the next few weeks

**Comment by skidipap** [comment]
i'm gonna make PR to this

**Comment by gautamkrishnar** [comment]
Thanks @skidipap it would be great.

**Comment by DeepSpace2** [comment]
Any progress on this? Or can I take a stab at it?

**Comment by gautamkrishnar** [comment]
@DeepSpace2 nothing yet

**Comment by gautamkrishnar** [comment]
@suvhotta thanks a lot, yep you are right. Hope you are planning to use https://github.com/pygments/pygments ?

**Comment by suvhotta** [comment]
@gautamkrishnar @hedyhli I tried formatting the question description part. The formatting is being done, however in the display part since we are converting it into a unicode text, it simply prints the format as any regular text, instead of displaying in the output. I've attached a screenshot of the issue, for your reference. 
Any ideas on how to solve this one?
![Screenshot 2021-11-02 at 7 08 01 PM](https://user-images.githubusercontent.com/16841978/139857899-34ce1d00-fd3c-4789-9f07-45ea26bbc493.png)

**Comment by tupokraju** [comment]
Hey, is this still open I am ready to work on it

**Comment by gautamkrishnar** [comment]
@suvhotta sorry for the late reply. I am only seeing your comment just now. @tupokraju if @suvhotta abandoned this, you can pick it up.

**Comment by mansidw** [comment]
Hey @gautamkrishnar, is this issue still open? Would like to give it a go.
